src/main/groovy/org/labkey/gradle/plugin/extension/UiTestExtension.groovy

```diff
@@ -23,7 +24,8 @@ import org.labkey.gradle.util.PropertiesUtils
 
 class UiTestExtension
 {
-    String propertiesFile = "test.properties"
+    final String propertiesFileName = "test.properties"
+    final String propertiesDistFileName = "test.properties.dist"
```

Comment:
I'm confused about the purpose of the `.dist` file.  I had thought it was going to be a distribution definition, but apparently that's not it.  Is it a secondary set of properties?  Why do we need/want that?  Why do we believe this file will be there when the properties file won't be?  When will it be there?  A file rename may be in order, but at least a comment or two about when, why, and what this file is would be helpful.

Reply:
My intention is that the dist file is checked in to source control and folks don't touch it (similar to our intellij config templates). The existing `test.properties` will be deleted and added to `.gitignore` in testAutomation. `UiTestExtension` will generate `test.properties` automatically if it doesn't exist and folks can put their local configurations in there without cluttering their changelist.

Reply:
I'm still not wild about the use of `dist` here just because it means something distinctly different elsewhere. I don't insist, but would `template` or `config` in place of `dist` be suitable?  In any case, comments on these two file names seem like they would be useful (e.g., for `propertiesDistFileName` it could be "the file checked in to VCS that is copied to propertiesFileName as part of the build" and for `propertiesFileName`, "the name of the file actually used in running tests").

Reply:
I was following some pattern I've see elsewhere (TeamCity comes with some default config files named with '.dist'). But you're correct, '.template' would be more consistent with elsewhere in LabKey; I'll make the switch.